addons/Text_To_Speech/src/presenter.js

```diff
@@ -495,6 +495,32 @@ function AddonText_To_Speech_create() {
         return splitText;
     }
 
+    presenter.removeUnnecessaryPunctuationMarks = function (texts) {
+        texts.forEach(text => {
+            const splittedWords = text.text
+                .replaceAll(" \/", ",")
+                .split(" ");
+
+            const newSentence = [];
+            splittedWords.forEach(word => {
+                if (word.trim().length) {
+                    const matchAfterLetter = word.match(/(\w\W)[.,?!]+/); // regex captures at least 2 special characters after word
```

Comment:
nie łapie przynajmniej 2 specjalnych znaków. Łapie przynajmniej 1 specjalny znak

Reply:
Proszę sprawdź przypadek `dwad ?!`. Teoretycznie powinno to coś psuć, zaś z drugiej strony na lekcji testowej jest jakby ok.